### DISCUSSION TITLE
Big Sur MacOS and Appcelerator Not Playing Nicely

### DISCUSSION THREAD

**Discussion by emptybox** [appcelerator/titanium_mobile#13291] [General]
Is anyone running Titanium on MacOS Big Sur? I have a few customer apps that need to be updated while I move over to Flutter unless I can get Titanium up and running. The fix here (https://stackoverflow.com/questions/62647625/not-able-to-run-eclipse-on-macos-big-sur) does not work for me. I can actually get past the JRE error. When I get to the launch screen, the prompt to determine default workspace shows up and then it freezes. My last resort is to reset my comp and install the shipped software which is Mavericks. Any help on this would be greatly appreciated.

**Here is my setup when running 'appc info' in the terminal:**

Operating System
  Name                        = macOS
  Version                     = 11.6
  Architecture                = 64bit
  CPUs                      = 8
  Memory                      = 16.0GB

Node.js
  Node.js Version             = 12.22.6
  npm Version                 = 8.1.4

Appcelerator CLI
  Installer                   = 6.0.0
  Core Package                = 9.0.1

Titanium CLI
  CLI Version                 = 5.3.2
  node-appc Version           = 1.1.2

I have SDK v9.2.2.GA installed as the latest.

**Comment by phillipmovista** [comment]
Yes, I have 10.1.1.GA running on Big Sur.

I did a variety of things like:
```
Install the Titanium CLI via [sudo] npm i -g titanium
Install the Alloy CLI via [sudo] npm i -g alloy
Build and run the app with titanium build -f -p [yourplatform]
```

as well as

```
export JAVA_HOME=`/usr/libexec/java_home -v 11.0.12`
titanium build -f -p android --log-level info
```

for Android specific.

Oh yeah, this stops using `appc` and just uses `titanium`

Can you share more of your info?

**Reply by emptybox** [reply]
Thank you for your help with this!

**Comment by m1ga** [comment]
Do you still need to use 9.2.2? And when you say "Workspace" do you still use Appcelerator Studio? Don't do that :smile: It's been abandoned for a long time now! Use VSCode or Atom with the Appc packages. way easier/faster/better!

If you can or want to update to the latest SDK: node 14 (or 16 but that will currently show a warning but works fine for me), jdk11, and the npm parts mentioned above or from: https://github.com/m1ga/from_zero_to_app/blob/master/installation.md

**Reply by emptybox** [reply]
This worked! Thank you! I can run from CLI beautifully.
